python_modules/dagster-test/dagster_test/graph_job_op_toys/error_monster.py

```diff
@@ -97,94 +95,100 @@ def _act_on_config(solid_config):
+    if context.op_config["return_wrong_type"]:
         return string + string
 
     return int(string)
 
 
-@pipeline(
-    description=(
-        "Demo pipeline that enables configurable types of errors thrown during pipeline execution, "
-        "including solid execution errors, type errors, and resource initialization errors."
-    ),
-    mode_defs=[
-        ModeDefinition(
-            name="errorable_mode",
-            resource_defs={
-                "errorable_resource": define_errorable_resource(),
-                "io_manager": errorable_io_manager,
-            },
-        ),
-    ],
-    preset_defs=[
-        PresetDefinition.from_pkg_resources(
-            "passing",
-            pkg_resource_defs=[("dagster_test.toys.environments", "error.yaml")],
-            mode="errorable_mode",
-        )
-    ],
-    tags={"monster": "error"},
+@graph(
+    description="Demo graph that enables configurable types of errors thrown during job execution, "
+    "including op execution errors, type errors, and resource initialization errors."
 )
 def error_monster():
     start = emit_num.alias("start")()
     middle = num_to_str.alias("middle")(num=start)
     str_to_num.alias("end")(string=middle)
 
 
+config = {
+    "resources": {"errorable_resource": {"config": {"throw_on_resource_init": False}}},
+    "ops": {
+        "end": {"config": {"return_wrong_type": False, "throw_in_op": False}},
+        "middle": {"config": {"return_wrong_type": False, "throw_in_op": False}},
+        "start": {"config": {"return_wrong_type": False, "throw_in_op": False}},
+    },
+}
+
+error_monster_passing_job = error_monster.to_job(
+    resource_defs={
+        "errorable_resource": define_errorable_resource(),
+        "io_manager": errorable_io_manager,
+    },
+    config=config,
+    tags={"monster": "error"},
+    executor_def=in_process_executor,
+)
+
+
 if __name__ == "__main__":
-    result = execute_pipeline(
-        error_monster,
-        {
-            "solids": {
-                "start": {"config": {"throw_in_solid": False, "return_wrong_type": False}},
-                "middle": {"config": {"throw_in_solid": False, "return_wrong_type": True}},
-                "end": {"config": {"throw_in_solid": False, "return_wrong_type": False}},
+    result = error_monster.to_job(
+        config={
+            "ops": {
+                "start": {"config": {"throw_in_op": False, "return_wrong_type": False}},
+                "middle": {"config": {"throw_in_op": False, "return_wrong_type": True}},
+                "end": {"config": {"throw_in_op": False, "return_wrong_type": False}},
             },
             "resources": {"errorable_resource": {"config": {"throw_on_resource_init": False}}},
         },
-    )
+        tags={"monster": "error"},
+        resource_defs={
+            "errorable_resource": define_errorable_resource(),
+            "io_manager": errorable_io_manager,
+        },
+        executor_def=in_process_executor,
```

Comment:
This shouldn't be necessary when using `execute_in_process`. Was it using the multiprocess executor? If so, that's def a bug.

Reply:
Ah yeah you're right, this isn't necessary.